config/settings.yml

```diff
@@ -16,3 +16,5 @@ redis_url: compliance-redis.compliance-ci.svc.cluster.local:6379
 slack_webhook: 'this is set through a env var in Openshift and not shared for security reasons'
 disable_rbac: 'false'
 async: true
+features:
+  parse_external_reports: false
```

Comment:
I thought this might be a good time to use a feature flag in the backend. For the 5 weeks or so we'll have to support external reports while not parsing them, it will be useful to allow devs and QEs to still parse external reports.

Reply:
:+1: